uber/decorators.py

```diff
@@ -485,3 +485,20 @@ def __call__(self, func):
         redirect_func = create_redirect(self.url or '../' + get_module_name(func) + '/' + func.__name__)
         setattr(root, self.alias_name or func.__name__, redirect_func)
         return func
+
+
+def id_required(func):
+    @wraps(func)
+    def check_id(self, **params):
+        message = "ID NOT FOUND"
```

Comment:
"ID NOT FOUND" makes sense to coders but probably doesn't make sense to someone who's not technical.  A better message would probably be something like "You weren't found in our database."

Though stepping back a minute, the main situation where this can occur is when someone hits Enter on the URL bar or otherwise copy/pastes a URL that doesn't actually have the ID.  An error message which reflects this and suggests how someone can rectify this would be helpful.